packages/gatsby-transformer-sqip/package.json

```diff
@@ -24,9 +24,8 @@
     "debug": "^4.3.4"
   },
   "peerDependencies": {
-    "gatsby": "^5.0.0-alpha-v5",
-    "gatsby-source-contentful": "^6.0.0-next",
-    "gatsby-transformer-sharp": "^4.0.0-next"
+    "gatsby": "^5.0.0-next",
+    "gatsby-transformer-sharp": "^5.0.0-next"
```

Comment:
this has `gatsby-source-contentful` removed and the plugin tries to import it in https://github.com/gatsbyjs/gatsby/blob/b2ab0921d124e23c73c9315fccdc161461f29714/packages/gatsby-transformer-sqip/src/extend-node-type.js#L139-L141

is that intential change to remove it?

Reply:
Ahh, I thought this was just a typo and not actually used...